package.json

```diff
@@ -14,35 +14,27 @@
   "author": "",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "bin": {
-    "payid-utils": "./dist/cli.js"
-  },
   "files": [
     "dist/*"
   ],
   "scripts": {
-    "preinstall": "npx npm-force-resolutions",
     "build": "rm -rf dist && tsc --project .",
     "lint": "eslint . --ext .ts --fix --max-warnings 0 && prettier --write '**/*.{md,json}'",
     "lintNoFix": "eslint . --ext .ts --max-warnings 0 && prettier --check '**/*.{md,json}'",
+    "prepare": "npm run build",
```

Comment:
Why do we have this command?

Reply:
I needed this in order to pull in a github branch as a dependency via npm. Since I need to be able to test this branch with the new cli project